xml/System/ArraySegment`1.xml

```diff
@@ -1017,16 +1020,10 @@
       </Parameters>
       <Docs>
         <param name="item">The object to add to the array segment.</param>
-        <summary>Adds an item to the array segment.</summary>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- This member is an explicit interface member implementation. It can be used only when the <xref:System.ArraySegment%601> instance is cast to an <xref:System.Collections.Generic.ICollection%601> interface.  
-  
- ]]></format>
-        </remarks>
-        <exception cref="T:System.NotSupportedException">The array segment is read-only.</exception>
+        <summary>Throws a <see cref="T:System.NotSupportedException" /> exception in all cases.</summary>
+        <returns>Throws a <see cref="T:System.NotSupportedException" /> exception in all cases.</returns>
+        <remarks>To be added.</remarks>
+        <exception cref="T:System.NotSupportedException">In all cases.</exception>
```

Comment:
It's already clear that an exception is thrown in all cases; better here would be the condition -- that an array segment cannot be expanded.

Reply:
That is what the style guide says for these cases: https://github.com/dotnet/dotnet-api-docs/wiki/Exceptions

I should add this information to the remarks as instructed here: https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Method